sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2Strategy.scala

```diff
@@ -245,6 +245,31 @@ object DataSourceV2Strategy extends Strategy with PredicateHelper {
       }.toArray
       DeleteFromTableExec(r.table.asDeletable, filters) :: Nil
 
+    case UpdateTable(r: DataSourceV2Relation, attrs, values, condition) =>
```

Comment:
@cloud-fan, do you think this should have a corresponding table capability, or is it okay because the table itself implements `SupportsUpdate`?

Reply:
In general we should always use table capability, but for delete/upadte, it's really weird to add capability as table must implement `SupportDelete/Update`, which is redundant to the capability.